### ISSUE TITLE
Minor: some style warnings are reported for comments that shouldn't

### ISSUE THREAD

**Issue by dcbaker** [PyCQA/pylint#856]
Pylint reports some warnings for comments, reporting these sort of warnings is silly unless they are comment specific (like two spaces between code and inline comments).

Contrived example:

``` python
import subprocess

dirs = subprocess.check_output(
    ['echo', '$PATH'],
# Convert to unicode and split on directory separators
).decode('utf-8').split(':')
```

In this case the comment will be marked as wrong hanging indent, which is silly since it applies to the line below, and makes sense to be dedented at that level.

**Comment by PCManticore** [comment]
Thanks for reporting this issue! It should probably be easy to filter out comments from the formatter checker with regard to these kind of patterns.

**Comment by mbyrnepr2** [comment]
I can't reproduce this on the current pylint; @DanielNoord perhaps you could be a second pair of eyes and this issue can be closed?

**Comment by Pierre-Sassoulas** [comment]
We removed wrong-hanging-indent from pylint à while ago.
